### PR TITLE
[Please review me] Operation: Funky Nostaligia 2. Changes charliestation medkit icon, and changes heirloom item for MDs to an old medkit.

### PR DESCRIPTION
**Pull request by trollbreeder**
Funky nostaligia 1 failed because i'm a clown

## About The Pull Request

1. Changes charliestation ancient medkits (those with bruise packs and ointment) to use the old medkit icon.
2. Adds a new family heirloom to MDs, PMs and CMOs: an old, empty medkit. It's empty because of assistants who kept breaking into medbay storage to heal themselves. This also removes the bodybag and stethoscope heirloom from MDs, CMOs and PMs in place of this medkit.

What's different this time? Well, i made this from a (hopefully recent enough) master and there's no horrific merge conflict frankenstein abominations this time. **Oh, and MDs, CMOs and PMs' heirlooms have all been removed in place of this ancient medkit.**

## Why It's Good For The Game

1. When i think of charliestation, i think of so old that the designs are from 2553. These ancient medkits use the normal medkit icon and that was a lost opportunity to me, so that is now rectified.
2. I would've expected more from the heirloom of MDs, who had to cope with a lot of changes, but apparantly not, just a body bag and a stethoscope. That's now rectified and MDs can now bring a piece of history along with them.

## Changelog
:cl:
add: Nanotrasen has asked to return all first-aid kits produced before 2559 as they've been recalled due to potential risk of lead acetate poisoning.
add: Medical doctors have been seen emotionally attached to these outdated first-aid kits.
del: Nanotrasen reports fewer cases of necrophillia, as medical doctors no longer feel any emotions towards body bags.
del: Nanotrasen also reports that medical doctors are no longer attached to 21st century heart listening devices.
imageadd: Ancient medkits are now properly ancient in design.
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -114,7 +114,7 @@
 	generate_items_inside(items_inside,src)
 
 /obj/item/storage/firstaid/ancient
-	icon_state = "firstaid"
+	icon_state = "oldifrstaid" // Alan please correct this typo because i have no idea how merges work
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/storage/firstaid.dm]: time to learn sweetie!

**Comment by Fikou** [comment]
nostaligia

File: code/game/objects/items/storage/firstaid.dm (hunk 1 above)
Comment (trollbreeder) [reply code/game/objects/items/storage/firstaid.dm]: hooray i've learned

**Comment by trollbreeder** [comment]
hooray, i've figured out how to update my local

**Comment by trollbreeder** [comment]
ahh bugger more conflicts

**Comment by trollbreeder** [comment]
okay nope

**Comment by trollbreeder** [comment]
DNM while i revert

**Comment by trollbreeder** [comment]
T-that should f-fix it right?! H-hopefully????

**Pull request merged**